src/content/en/fundamentals/security/credential-management/save-forms.md

```diff
@@ -140,10 +137,14 @@ or proceed to the personalized page.
       e.preventDefault();
 
       // Try sign-in with AJAX
-      fetch(/'signin', {
+      let form = new FormData();
+      form.append('email', c.id);
+      form.append('password', c.password);
+
+      return fetch('/signin', {
         method: 'POST',
-        body: new FormData(e.target),
-        credentials: 'include'
+        credentials: 'include',
+        body: form
       }).then(res => {
```

Comment:
Hey,

So bit confused here. 

Should I go back to original version? In which case, I should roll back changes to earlier code that is part of the full sample.

I made a choice to go back to my original code changes-- these may still be wrong. But see what you think?

Would help if you just copied full code example that should be here, if it is still incorrect.